Dockerfile.dapper

```diff
@@ -10,7 +11,15 @@ RUN wget -O - https://download.docker.com/linux/debian/gpg | apt-key add - && \
     cat /etc/apt/sources.list
 
 RUN apt-get update && \
-    apt-get install -y docker-ce bash git jq
+    apt-get install -y bash git jq
+
+RUN if [ "${ARCH}" != "s390x" ]; then \
```

Comment:
try and stay consistent on this if/else logic. In loglevel you had it reversed https://github.com/rancher/loglevel/pull/9/files#diff-3598233dce47ce9c207a77a9f23394d45235397dd66d7dd9de909487e1bd8e8aR27-R33 and i prefer that one personally as an else should be "everything else gets this version" where s390x is the exception

Reply:
okie.